connector/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -51,11 +51,14 @@ object SchemaConverters {
    *
    * @since 2.4.0
    */
+  def toSqlType(avroSchema: Schema, useStableIdForUnionType: Boolean): SchemaType = {
+    toSqlTypeHelper(avroSchema, Set.empty, useStableIdForUnionType)
+  }
   def toSqlType(avroSchema: Schema): SchemaType = {
-    toSqlTypeHelper(avroSchema, Set.empty, AvroOptions(Map()))
+    toSqlTypeHelper(avroSchema, Set.empty, AvroOptions(Map()).useStableIdForUnionType)
```

Comment:
Could we avoid the usage `AvroOptions(Map()).useStableIdForUnionType`?

Reply:
sure, jut completely remove this method now.